hannakim91/travel-tracker
# Features/final touches

# Travel Tracker Pull Request

### What’s this PR do?  
Final updates before project deadline:
- Add logo to log-in page
- Find appropriate color scheme for color-blindness
- Style elements on log-in/modal/dashboard
- Display trip cards in order of most future to past trips and so newest trip comes up at front of trip cards area (until refresh -- it may get sorted in line if they have more further in the future trips than trip just booked)
- Add image to trip cards
- Refactor to make more use of SASS
- Run linter to clean up codebase
### Where should the reviewer start?  
Most changes are in CSS directory, but some necessary changes to classes/tests, index.js, index.html, domUpdates.js
### How should this be manually tested?  
Run `npm start` and open localhost:8080 in different browsers - checked to see they look pretty good on Chromium-based browsers (my defualt), Mozilla Firefox, and Safari
### Any background context you want to provide?  

### What are the relevant tickets?  
Resolves #75;
Resolves #55;
Resolves #30;
Resolves #28; 
Resolves #18;
Resolves #76;
Resolves #66;
Resolves #74;
Resolves #60;
### Screenshots (if appropriate)  

### Questions:
